WcaOnRails/app/models/competition.rb

```diff
@@ -699,15 +699,15 @@ def psych_sheet_event(event)
 
     prev_registration = nil
     registrations.each_with_index do |registration, i|
-      if event.recommended_format.sort_by == :single
+      if sort_by == 'single'
```

Comment:
It actually didn't work before. This part of the code is used to tell if the already sorted competitors are tied. The `event.recommended_format.sort_by == :single` always failed and hence all ties were computed by the average_rank (although the sorting was already performed correctly). The main test looking at ties was done using average. When I added a test for single it failed. So I've now changed the comparison and it now works.